dcc-portal-ui/app/scripts/genes/js/genes.js

```diff
@@ -39,6 +39,64 @@
 (function () {
   'use strict';
 
+  var module = angular.module('app.common');
+  
+  // @bob what should the module be called? - chang  
+  module.directive('fixScroll', function ($timeout) {
+    /**
+     * Our function for keeping the page on the current section. 
+     */
+    function scrollToCurrent () {
+      var target = jQuery(window.location.hash);
+      
+      // We do not want to immediately scroll away from the controls on page load. 
+      // Timeout of zero is used to ensure scroll happens after render. 
+      $timeout(function() {
+          jQuery('body,html').stop(true, true);
+          var offset = jQuery(target).offset();
+          var to = offset.top - 40;
+          jQuery('body,html').animate({scrollTop: to}, 200); 
```

Comment:
Question: Is the timeout needed now (also, would cache the jquery `jQuery('body,html')` expression)?